accumulo-store/src/main/java/gaffer/accumulostore/utils/AddUpdateTableIterator.java

```diff
@@ -17,14 +17,23 @@
 package gaffer.accumulostore.utils;
 
 import gaffer.accumulostore.key.exception.IteratorSettingException;
+import gaffer.data.elementdefinition.schema.DataSchema;
+import gaffer.data.elementdefinition.schema.exception.SchemaException;
+import gaffer.graph.Graph;
 import gaffer.accumulostore.AccumuloStore;
 import gaffer.store.StoreException;
+import gaffer.store.StoreProperties;
+import gaffer.store.schema.StoreSchema;
+
 import org.apache.accumulo.core.client.AccumuloException;
 import org.apache.accumulo.core.client.AccumuloSecurityException;
 import org.apache.accumulo.core.client.IteratorSetting;
 import org.apache.accumulo.core.client.TableNotFoundException;
 import org.apache.accumulo.core.iterators.IteratorUtil.IteratorScope;
 
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.EnumSet;
 
 public class AddUpdateTableIterator {
```

Comment:
It would be helpful to have a javadoc explaining that this class has a main method and when it should be used